lld/test/ELF/arm-thumb-thunk-v6m-noshort.s

```diff
@@ -0,0 +1,57 @@
+// REQUIRES: arm
```

Comment:
The test passes even without the code change. Is that expected?

Reply:
Apologies it looks like I forgot to add FileCheck to the objdump for v6-m. Now done, which revealed a typo induced by fumbling a tmux switch buffer.

Tested that this now passes with the patch, and doesn't with an existing lld.